vllm/model_executor/models/llama.py

```diff
@@ -87,14 +87,17 @@ def __init__(
             quant_config=quant_config,
             prefix=f"{prefix}.down_proj",
         )
-        self.use_fp8 = isinstance(quant_config, Fp8Config)
+        self.use_fp8 = isinstance(quant_config, Fp8Config) \
```

Comment:
Here is controls fused kernels rather than datatypes. Worth checking if it works on Navi and not disabling